packages/tenant-readmodel-writer/test/converterV1.ts

```diff
@@ -94,6 +94,7 @@ export function toAttributeV1(input: TenantAttribute): TenantAttributeV1 {
 
 export function toTenantMailV1(mail: TenantMail): TenantMailV1 {
   return {
+    id: mail.id ?? undefined,
```

Comment:
the default value is the sha256 of the mail address ([ref](https://github.com/pagopa/interop-be-tenant-management/blob/1.0.x/src/main/scala/it/pagopa/interop/tenantmanagement/model/persistence/serializer/v1/protobufUtils.scala#L79))

Reply:
I think it's already handled [here](https://github.com/pagopa/interop-be-monorepo/blob/289c2a6298b153989f2a289870f3e36633b52b58/packages/models/src/tenant/protobufConverterFromV1.ts#L62-L64).